c/parallel/src/util/tuning.h

```diff
@@ -0,0 +1,13 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
```

Comment:
Is this correct? Isn't this the C parallel library?

Reply:
I'm not really sure, this is the license used in the other files in the same directory